### PR TITLE
Using notion for documentation

### PR DESCRIPTION
**Pull request by jessyrosy**

### CODE REVIEW

**Comment by tonyalaribe** [comment]
1. Can you write more text in the article? It feels too sparse.
2. Could you actually show how to write API documentation? Your article talks about API documentation, but your walkthrough is for a regular random documentation page, not API documentation. If you want this article to be about regular documentation, that's also fine, but make it clear in the article.

File: content/blog/using-notion-for-api-documentation/index.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+![Type of new template](type-of-new-template.png)
+
+And when you are done filling out the form, you can go ahead to start documenting every bit of that process to guide your users accurately.
+![New work ready](create-new-template.png)
+
+## Automation is fast and more effective
+An [API Documentation](../top-8-api-documentation-tools-for-developers) is an essential component of each API project. The documentation should accurately reflect each step of the process and should be easy to understand for new users; however, if it is written with no form of orderliness or in a hurry, it may end up being jumbled
+
+Notion is totally perfect as an app for maintaining a workflow environment but has a bit lapses as regards documenting. The important thing to note is that not all developers have enough time on their plate to go about the notion process so they will prefer automatic documentation tools like APIToolkit to automate theri documentation and keep it up to date. [Click ](../updates-october-2022) to learn more about APIToolkit.
```
Comment (tonyalaribe) [inline content/blog/using-notion-for-api-documentation/index.md:47]:
```suggestion
Notion is totally perfect as an app for maintaining a workflow environment but has a bit of lapses as regards documenting. The important thing to note is that not all developers have enough time on their plate to go about the notion process so they will prefer automatic documentation tools like APIToolkit to automate theri documentation and keep it up to date. [Click to learn more about APIToolkit](../updates-october-2022).
```

**Comment by tonyalaribe** [comment]
image
Is this intended? I think the markdown is malformed.

**Comment by jessyrosy** [comment]
Yes, I can add more text to the article. I will update the article to
include a section on how to write API documentation. I feel like since the
focus is on Notion we should just focus on notion and add a link to how to
write API documentation.

**Review by tonyalaribe — APPROVED** [review]

**Pull request merged**